pkg/services/pluginsintegration/pluginsintegration.go

```diff
@@ -189,6 +189,8 @@ func CreateMiddlewares(cfg *setting.Cfg, oAuthTokenService oauthtoken.OAuthToken
 
 	middlewares = append(middlewares, clientmiddleware.NewHTTPClientMiddleware())
 
+	middlewares = append(middlewares, clientmiddleware.NewPerconaForwarderHTTPClientMiddleware())
```

Comment:
Please add a `@PERCONA` comment here, helps with seeing PMM modifications during grafana upgrade.

Reply:
Added here and to the other file